src/app/routes/utils/regex/utils/index.js

```diff
@@ -11,12 +11,13 @@ const tvMasterBrandRegex = 'bbc_[a-z]+_tv';
 const errorCodeRegex = '404|500';
 const idxRegex = 'persian/afghanistan|ukrainian/ukraine_in_russian';
 const brandEpisodeRegex = 'tv|tv_programmes';
+const disciplineRegex = '/[a-z0-9-_]{1,}';
 
 const getServiceRegex = services => services.join('|');
 
 export const getArticleRegex = services => {
   const serviceRegex = getServiceRegex(services);
-  return `/:service(${serviceRegex})/:local(${articleLocalRegex})/:id(${idRegex}):variant(${variantRegex})?:amp(${ampRegex})?`;
+  return `/:service(${serviceRegex})?:discipline(${disciplineRegex})?/:local(${articleLocalRegex})/:id(${idRegex}):variant(${variantRegex})?:amp(${ampRegex})?`;
```

Comment:
can we change this var? I was difficult to for me understand in the context of simorgh itself. If it's just sport, might we call it `sportDiscipline`? Or even just a comment by the `disciplineRegex` might suffice. I just fear we'll forget what this is in a year's time!

Reply:
Yes this makes sense. I'm pretty sure this is just for Sport. I have updated to be `discipline(${sportDisciplineRegex}) ` since this follows the pattern of `local(${articleLocalRegex)`.